dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,376 @@
+                            "title": "LksG",
+                            "text": "101",
+                            "date": "14th of August"
+                        },
+                        {
+                            "icon": "/static/icons/Explore.svg",
+                            "title": "Pathway to Paris",
+                            "text": "101",
+                            "date": "Starting soon!"
+                        }
+                    ]
+                },
+                {
+                    "title": "Come together",
+                    "text": [
+                        "Collaborate on datasets ",
+                        "to share the costs."
+                    ],
+                    "image": [
+                        "Join now!"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "/static/icons/Face--add.svg",
+                            "title": "10",
+                            "text": "Companies joining every day"
+                        },
+                        {
+                            "icon": "/static/icons/3d-mpr-toggle.svg",
+                            "title": "9000",
+                            "text": "European Companies"
+                        },
+                        {
+                            "icon": "/static/icons/Certificate.svg",
+                            "title": "3",
+                            "text": "Active Campaigns"
+                        },
+                        {
+                            "icon": "/static/icons/Area.svg",
+                            "title": "5",
+                            "text": "Frameworks"
+                        }
+                    ]
+                },
+                {
+                    "title": "Join a campaign",
+                    "text": [
+                        "Join a campaign to",
+                        " to get the missing ",
+                        "data.",
+                        "The Proof of Concepts are a simple way for you to get better data coverage and improved data quality, while testing Dataland."
+                    ],
+                    "cards": [
+                        {
+                            "title": "Pathway to Paris",
+                            "text": "Explanatory text goes here",
+                            "date": "14th of August"
+                        },
+                        {
+                            "title": "LksG",
+                            "text": "Explanatory text goes here"
+                        },
+                        {
+                            "title": "EU Taxonomy",
+                            "text": "Explanatory text goes here"
+                        },
+                        {
+                            "title": "SFDR",
+                            "text": "Explanatory text goes here"
```

Comment:
Explanations should be added before merging to main.

Reply:
I am still waiting for the final text

Reply:
Ok. Do you know who is responsible fro providing it to you?

Reply:
I take the content from a Figma file which is updated by our UX designer after aligning with PWC and D-fine

Reply:
Keeping this open as "wait-reminder".

Reply:
Closing this because now explanations are in place.